diff --git a/node_modules/@lerna/version/lib/update-lockfile-version.js b/node_modules/@lerna/version/lib/update-lockfile-version.js
index 1975c11..16b94b6 100644
--- a/node_modules/@lerna/version/lib/update-lockfile-version.js
+++ b/node_modules/@lerna/version/lib/update-lockfile-version.js
@@ -16,6 +16,12 @@ function updateLockfileVersion(pkg) {
     if (obj) {
       obj.version = pkg.version;
 
+      // Patch for lock file version not updating issue
+      // Refer - https://github.com/lerna/lerna/pull/3091
+      if (obj.packages && obj.packages[""]) {
+        obj.packages[""].version = pkg.version;
+      }
+
       return writeJsonFile(lockfilePath, obj, {
         detectIndent: true,
         indent: 2,
